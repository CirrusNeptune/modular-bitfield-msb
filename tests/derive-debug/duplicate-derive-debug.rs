use modular_bitfield_msb::prelude::*;

#[bitfield]
#[derive(Debug)] #[derive(Debug)]
pub struct SignedInt {
    sign: bool,
    value: B31,
}

fn main() {}
