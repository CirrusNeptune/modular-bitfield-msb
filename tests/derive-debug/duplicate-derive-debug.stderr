error: encountered duplicate `#[derive(Debug)]` attribute: duplicate set to true
 --> $DIR/duplicate-derive-debug.rs:5:10
  |
5 | #[derive(Debug)]
  |          ^^^^^

error: previous `#[derive(Debug)]` parameter here
 --> $DIR/duplicate-derive-debug.rs:4:10
  |
4 | #[derive(Debug)]
  |          ^^^^^
